src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/Wix_3.11.1_VS2017_Bootstrapper.exe.sarif

```diff
@@ -784,8 +784,6 @@
       "tool": {
         "driver": {
           "name": "testhost",
-          "organization": "Microsoft Corporation",
-          "product": "Microsoft.TestHost",
```

Comment:
>Microsoft [](http://example.com/codeflow?start=22&length=9)

how did u generate this tests?
this should not appear.

Reply:
I think this is caused by, I see there shows a sub mudule update and I update it.

Reply:
re-ran the UpdateBaselines.ps1, same result, so this should be caused by the new version of sub module